src/katgpucbf/fgpu/send.py

```diff
@@ -316,7 +316,7 @@ def make_streams(
         # Adding len(endpoints) to accommodate descriptors sent for each substream
         max_heaps=n_data_heaps + len(endpoints),
     )
-    streams: list["spead2.send.asyncio.AsyncStream"]
```

Comment:
Do you remember why we had to insert quotes in the first place?

Reply:
That type is a fiction of the spead2 type annotations and doesn't exist at runtime. Python is a little inconsistent about where one needs to quote such fictional types. In function signatures you need the quotes as otherwise it tries to look up the type at runtime, but for local variables in a function you don't need it (I think that's because you can interrogate a function's signature at runtime using the `inspect` module). So I don't think these quotes were ever needed, but I got into the habit of putting them in rather than try to remember where they're required.